packages/api-explorer/src/ResponseExample.jsx

```diff
@@ -257,9 +278,7 @@ class ResponseExample extends React.Component {
               })}
             </div>
           </span>
-        )}
-
-        {(examples.length === 0 || (!hasExamples && result === null)) && (
```

Comment:
Removing this `result` check from here since `examples.length` now takes precedence over this as we're now handling responses w/o examples.